polylabel.js

```diff
@@ -65,10 +73,10 @@ function polylabel(polygon, precision, debug) {
 
```

Comment:
This bit is sort of a mystery to me `if (cell.max - bestCell.d <= precision) continue;`.

I wonder if the `bestCell.d` should be replaced here by `fitness(bestCell)`. I don't understand all the logic behind `max`, so I left this as-is. Seems to work fine like this.

Reply:
This test corresponds to point 4 of the [algorithm](https://github.com/mapbox/polylabel#how-the-algorithm-works).

Reply:
If it helps, my interpretation of that line is that `cell.max` provides the best possible value for distance from the polygon edge, and if the `bestCell.d` is at least within `precision` of `cell.max` then a better solution does not exist within the selected `cell`, and we can move on through the priority queue.